src/Metrics.php

```diff
@@ -107,7 +107,7 @@ public function getMetricsTimeline($since = null, string $uuid = null, int $coun
      * @return bool|mixed
      *
      */
-    public function getMetricTimeline( string $metricID, $since = null, $uuid = null, int $count = null, string $sort = null)
+    public function getMetricTimeline( $metricID, $since = null, $uuid = null, $count = null, $sort = null)
```

Comment:
```suggestion
    public function getMetricTimeline( $metricID, $since = null, $uuid = null, $count = null, $sort = null )
```

Reply:
there are a lot of formatting inconsistencies but that's out of scope for this quick fix